libs/vkd3d/device.c

```diff
@@ -4771,9 +4822,22 @@ static void d3d12_device_caps_init_shader_model(struct d3d12_device *device)
 
         /* SM 6.3 adds:
          * https://github.com/microsoft/DirectXShaderCompiler/wiki/Shader-Model-6.3
-         * Ray tracing
+         * Ray tracing (lib_6_3 multi entry point targets).
          */
 
+        /* Can be overridden with FL override for now.
+         * Enable SM 6.3 when RT is somewhat working. */
+#if 0
+        if (device->d3d12_caps.max_shader_model == D3D_SHADER_MODEL_6_2 &&
+            device->device_info.ray_tracing_pipeline_features.rayTracingPipeline &&
+            device->vk_info.KHR_spirv_1_4)
+        {
+            /* SPIR-V 1.4 is required for lib_6_3 since that is required for RT. */
+            device->d3d12_caps.max_shader_model = D3D_SHADER_MODEL_6_3;
+            TRACE("Enabling support for SM 6.3.\n");
+        }
+#endif
```

Comment:
Is DXR a hard requirement for D3D_SHADER_MODEL_6_3 and up?

Reply:
Somewhat unclear, it's tied to DXR somehow, but it's mostly about the lib_6_3 target. It's possible to compile non-RT shaders with lib_6_3.

It should be fine to enable it.

Reply:
Not sure how I feel about that commit now ... An alternative is to just drop the commit. FL 12.2 override takes care of it as well for dev purposes.